### PR TITLE
Migration page index of Mocktail, set css and js

### PR DESCRIPTION
**Pull request by chouaibj**

### CODE REVIEW

**Review by joelmayemba — APPROVED** [review]

File: assets/js/filters.js
Comment (Benja-san) [inline assets/js/filters.js:1]: rename this file mocktailsFilter for better understanding of the file

File: assets/styles/pages/mocktails.css [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+    border: none;
+}
+
+/* 
+---------------------------------------------
+Responsive Style
+--------------------------------------------- 
+*/
+@media screen and (max-width: 450px) {
+    .contact-page #contact-form {
```
Comment (Benja-san) [inline assets/styles/pages/mocktails.css]: contact page styling has nothing to do in your mocktail css file

File: templates/mocktail/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                <li>
+                    <button class="btn filter-button active" data-filter="all">All</button>
+                </li>
+                {% for category in categories %}
+                    <li>
+                        <button class="btn filter-button" data-filter="{{ category['id'] }}">{{ category["name"] }}</button>
+                    </li>
+                {% endfor %}
+            </ul>
+            <div class="row mocktails-box">
```
Comment (Benja-san) [inline templates/mocktail/index.html.twig]: ul instead of div

File: templates/mocktail/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                {% for mocktail in mocktails %}
+                    <div class="col-lg-3 col-md-6 col-sm-6 align-self-center mb-30 properties-items col-md-6" data-tag="{{ mocktail['category_id'] }}">
```
Comment (Benja-san) [inline templates/mocktail/index.html.twig]: li instead of div

File: templates/mocktail/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                        <div class="item text-center" data-product-name="{{ mocktail.name }}">
```
Comment (Benja-san) [inline templates/mocktail/index.html.twig]: article instead of div

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: templates/mocktail/index.html.twig [hunk 5]
```diff
@@ -1,57 +1,69 @@
...
-                    <button class="btn filter-button active" data-filter="all">All</button>
-                </li>
-                {% for category in categories %}
+{% extends 'base.html.twig' %} {% block title %} {{ parent() }}
+{{ pageName }} {% endblock %} {% block stylesheet %}
+<link rel="stylesheet" href="assets/styles/pages/mocktails.css" />
+{% endblock %} {% block javascript %}
+<script src="assets/js/mocktailsFilter.js"></script>
+{% endblock %} {% block content %}
+<div class="section mocktails">
```
Comment (Benja-san) [inline templates/mocktail/index.html.twig]: use main tag instead of div tag

File: templates/mocktail/index.html.twig [hunk 6]
```diff
@@ -1,57 +1,69 @@
...
+                <button
+                    class="btn filter-button"
+                    data-filter="{{ category['id'] }}"
+                >
+                    {{ category["name"] }}
+                </button>
+            </li>
+            {% endfor %}
+        </ul>
+        <article class="row mocktails-box">
```
Comment (Benja-san) [inline templates/mocktail/index.html.twig]: section instead oif article, you will display a list of articles

Comment (Benja-san) [reply templates/mocktail/index.html.twig]: or you can use ul

File: templates/mocktail/index.html.twig [hunk 7]
```diff
@@ -1,57 +1,69 @@
...
+            {% for mocktail in mocktails %}
+            <div
```
Comment (Benja-san) [inline templates/mocktail/index.html.twig]: li instead of div tag

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: templates/mocktail/index.html.twig [hunk 8]
```diff
@@ -4,66 +4,70 @@
...
+                </li>
+                {% endfor %}
+            </ul>
+            <ul class="row mocktails-box">
+                {% for mocktail in mocktails %}
+                    <li
+                        class="col-lg-3 col-md-6 col-sm-6 align-self-center mb-30 properties-items col-md-6"
+                        data-tag="{{ mocktail['category_id'] }}"
+                    >
+                        <li
```
Comment (Benja-san) [inline templates/mocktail/index.html.twig]: li inside li

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by joelmayemba — APPROVED** [review]

File: src/Controller/MocktailController.php [hunk 9]
```diff
@@ -4,15 +4,22 @@
...
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\Routing\Attribute\Route;
+use Symfony\Component\Routing\Annotation\Route;
 
+#[Route('/mocktail', name: 'app_mocktail')]
 class MocktailController extends AbstractController
 {
-    #[Route('/mocktail', name: 'app_mocktail')]
+    #[Route('/', name: 'index')]
```
Comment (Benja-san) [inline src/Controller/MocktailController.php:12]: think about the underscore after "app_moktail" or before "index"

File: src/Controller/MocktailController.php [hunk 10]
```diff
@@ -4,15 +4,22 @@
...
     public function index(): Response
     {
-        return $this->render('mocktail/index.html.twig', [
+        return $this->render('index.html.twig', [
             'pageName' => "Nos Mocktails",
         ]);
     }
+
+    #[Route('/show', name: 'show')]
```
Comment (Benja-san) [inline src/Controller/MocktailController.php:20]: same for show route name

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Pull request merged**